setup.py

```diff
@@ -493,7 +493,7 @@ def do_setup():
             'tabulate>=0.7.5, <0.9',
             'tenacity==4.12.0',
             'termcolor==1.1.0',
-            'text-unidecode==1.3',
+            'text-unidecode==1.2',
```

Comment:
Don't think you wanted this change anymore

Reply:
yeah. Thanks .. will fix